### DISCUSSION TITLE
Help! Moving site from Netlify to GitHub Pages

### DISCUSSION THREAD

**Discussion by mmpadilla15** [thegetty/quire#662] [Get Help] [answered]
We've decided to move our epub from Netlify, where it is living now, to GitHub Pages as we found out our museum has a GitHub account that will accommodate it. I'm trying to follow the instructions in the documentation but it seems like there have been just enough changes over the last few months that things don't seem quite right. One example is that the recommended text editor, Atom, was sunsetted in December, so I had to find a new one. Here's what I've done successfully (I think) so far, in order:

- I've created a new repository in GitHub in my museum's account and I can see it
- From this [page](https://quire.getty.edu/docs-v0/site-deploy/), I'm following the instructions for Preview Your Project with GitHub Pages and I've done numbers 1-6

But when I get to number 7, I can't find how to commit and merge in either the desktop or the browser version. I've uploaded the two changed files, but I'm not sure they are in the correct place:
<img width="898" alt="Screenshot 2023-02-13 at 11 27 40 AM" src="https://user-images.githubusercontent.com/74433007/218529256-c59d2b94-bad4-4385-9363-6d06908c5c14.png">

Also, when I open Terminal and navigate to my project and enter `bin/deply.sh`, I get a message that says "no such file or directory."

Is someone able to help me get over these humps? Please and thank you!

**Comment by geealbers** [comment]
Hi @mmpadilla15. It looks from the screenshot above that you've followed [the steps to deploy your project with GitHub pages](https://quire.getty.edu/docs-v0/site-deploy/#github-pages), but in step 1 of that, there is a link that goes to instructions for getting your complete project files on GitHub first and I think maybe you missed that, or had some trouble with it and didn't realize.

The instructions are here: https://quire.getty.edu/docs-v0/github/#host-your-projects-code-on-github

From where you are now, I'd suggest deleting the repository your started on GitHub, and then starting with the above instructions to get your project published to a new repository. Once that's done, you can go back and redo the deploy instructions as you were doing them, and I think you'll have more luck.

**Reply by ACMAA** [reply]
@geealbers Thanks Greg! On this page: [https://quire.getty.edu/docs-v0/github/#host-your-projects-code-on-github](https://quire.getty.edu/docs-v0/github/#host-your-projects-code-on-github), I'm stuck on number 2. After I click Add Existing Repository and navigate to my files, I get a message that says "This directory does not appear to be a Git repository":
![Screenshot 2023-02-21 at 2 40 32 PM](https://user-images.githubusercontent.com/2111437/220454873-e7823777-77de-460a-a611-c79e614730b6.png)

When I click "Create a repository," I get this box:
![Screenshot 2023-02-21 at 2 41 15 PM](https://user-images.githubusercontent.com/2111437/220455178-e73b4bc9-ac20-4f63-a18e-8e4b1194f8ac.png)

I'm not sure what I'm supposed to do here and everything is grayed out.

Suggestions?

**Comment by Erin-Cecele** [comment, marked as answer]
Hi @ACMAA, it will probably be easiest for us to do a screen share together on a Zoom call. I sent you an email from the Quire email address to coordinate a time.

**Reply by Erin-Cecele** [reply]
Hi @ACMAA just wanted to check in and see how the move to GH Pages worked out. Are you all set now?

**Reply by mmpadilla15** [reply]
Hi @Erin-Cecele Yes! We are finally up and running on Github. I had to have out IT department do some DNS work to get it running. Yay!

**Reply by Erin-Cecele** [reply]
That's great! Thank you for confirming @mmpadilla15.
